apps/frontend/src/app/5_pages/StakePage/components/VestingStakesFrame/VestingStakesFrame.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  OrderDirection,
+  OrderOptions,
+  Pagination,
+  Paragraph,
+  Table,
+} from '@sovryn/ui';
+
+import { DEFAULT_STAKES_SIZE } from '../../../../../constants/general';
+import { useAccount } from '../../../../../hooks/useAccount';
+import { useBlockNumber } from '../../../../../hooks/useBlockNumber';
+import { translations } from '../../../../../locales/i18n';
+import { VestingContract } from '../../../../../utils/graphql/rsk/generated';
+import { COLUMNS_CONFIG } from './VestingStakesFrame.constants';
+import { generateRowTitle } from './VestingStakesFrame.utils';
+import { useGetVestingStakes } from './hooks/useGetVestingStakes';
+
+const pageSize = DEFAULT_STAKES_SIZE;
+
+export const VestingStakesFrame: FC = () => {
+  const { account } = useAccount();
+  const { value: block } = useBlockNumber();
+
+  const [page, setPage] = useState(0);
+
+  const [orderOptions, setOrderOptions] = useState<OrderOptions>({
+    orderBy: 'lockedUntil',
```

Comment:
```suggestion
    orderBy: 'createdAtTimestamp',
```

`lockedUntil` does not exist on `vestingContract`